### DISCUSSION TITLE
Hash check fail autofix

### DISCUSSION THREAD

**Discussion by AndrewSav** [ScoopInstaller/Scoop#5269] [General]
It seems there is a bot that can automatically fix failed hash checks, e.g <https://github.com/ScoopInstaller/Extras/issues/9859>. It became much less effective after the repo got the issue templates - no existing template accommodate this use case, and the existing bug template asks for a lot of information that the bot simply does not need.

I could be wrong but it seems to me that because of that there are issues that are not picked up by the bot, e.g [1](https://github.com/ScoopInstaller/Extras/issues/9750),[2](https://github.com/ScoopInstaller/Extras/issues/9858),[3](https://github.com/ScoopInstaller/Extras/issues/9790),[4](https://github.com/ScoopInstaller/Extras/issues/9776), etc

Can someone please create a new bug template that can work for this use case, and also update the scoop in-text to say something different from:

```
Please try again or create a new issue by using the following link and paste your console output:
https://github.com/lukesampson/scoop-extras/issues/new?title=openlens%406.2.3%3a+hash+check+failed
```

since a urls of this form no longer work.

I'd open an issue, but then again it is not clear where it should go, both scoop needs to be updated and bucket repositories issues template. Have no problem opening an issue, if you direct me where.

**Comment by niheaven** [comment]
No matter your issue content is, the title must not be changed, and the bot will correct it. The main reason is the title, and a new hash template will be added soon.

**Reply by AndrewSav** [reply]
Cheers
